res/controllers/Numark-Mixtrack-Platinum-scripts.js

```diff
@@ -15,8 +15,8 @@
 // should wheel be enabled on startup?
 var EnableWheel = true;
 
-// should we use the manual loop buttons as hotcue buttons 5-8?
-var UseManualLoopAsCue = false;
+// How should the manual loop pads behave by default?
+var UseManualLoopAs = 'stem'; // Options: 'stem', 'cue', 'normal' (or any other string for normal)
```

Comment:
How do you feel about using the [Settings API](https://github.com/mixxxdj/mixxx/wiki/midi%20scripting#settings-api) for this new one? Aso feel free to convert the other of course :D